include/psa/crypto_config.h

```diff
@@ -1675,7 +1665,7 @@
  *
  * Uncomment this macro to enable restartable ECC computations.
  */
-//#define MBEDTLS_ECP_RESTARTABLE
+// #define MBEDTLS_ECP_RESTARTABLE
```

Comment:
```suggestion
//#define MBEDTLS_ECP_RESTARTABLE
```

Reply:
Also the commit message that introduces this change seems unrelated with the change itself

Reply:
Thanks, I dropped the commit containing those changes as that is all it appeared to have.